packages/gafl-webapp-service/src/server.js

```diff
@@ -75,7 +75,6 @@ const init = async () => {
   const sessionCookieName = process.env.SESSION_COOKIE_NAME || SESSION_COOKIE_NAME_DEFAULT
 
   const sessionCookieOptions = {
```

Comment:
When will the session cookie expire?  It needs to have a ttl to align with the new PECR cookie guidance rules.

Reply:
At the moment has an expiry  of session. I do not know if this is compliant or not but basically the cache is doing the timing out. Its tricky otherwise because the cache ttl gets updated on every write whereas the cookie just runs down. I need to return to this I guess